src/components/modal/arbeidsliste/legg-til-arbeidslisteform.tsx

```diff
@@ -193,8 +193,8 @@ const mapDispatchToProps = (dispatch, props) => ({
         const {valgteBrukere} = props;
         const liste = arbeidsliste.map((elem, index) => ({
             fnr: valgteBrukere[index].fnr,
-            overskrift: elem.overskrift,
-            kommentar: elem.kommentar,
+            overskrift: elem.overskrift.length ? elem.overskrift : null,
```

Comment:
I `initialValues` er `overskrift`/`kommentar` satt til `''`. Siden vi her bare sjekker om `.length`-propertyen eksisterer så vil den alltid evaluere til true. Vet ikke helt om det var hensikten her. Uansett gir det kanskjer mer mening å sjekke på følgende:

```suggestion
            overskrift: elem.overskrift.length > 0 ? elem.overskrift : null,
```

Reply:
0 i length vil jo gi false?
![Screenshot 2023-11-13 at 11 02 12](https://github.com/navikt/veilarbportefoljeflatefs/assets/3892999/7fabeeba-f383-4f7a-b435-7f7447ffc875)

Men kan godt ha > 0 om du synes det er bedre kode? :)

Reply:
Godt poeng, `0` blir jo coerced til `false` - jeg tenkte litt feil der gitt 😬 
Men jeg synes fortsatt intensjonen til koden blir tydeligere dersom man sjekker på selve lengden `(> 0)` 🤔